manifests/profile/php73.pp

```diff
@@ -19,16 +19,14 @@
     architecture => $::os['architecture'],
   }
 
-  package { ['php-common',
-  'php-geoip',
-  'php-igbinary',
-  'php-imagick',
-  'php-memcached',
-  'php-msgpack',
+  package { ['php7.3-geoip',
+  'php7.3-igbinary',
+  'php7.3-imagick',
+  'php7.3-memcached',
+  'php7.3-msgpack',
   #  'php-pear',
-  'php-redis',
-  'php-xdebug',
-  'php-xml',
```

Comment:
Looks like maybe this one didn't make it over. What's the story for php-xml?

Reply:
are 'php-redis' and 'php-xdebug' necessary if they are specificed with 'php7.3-redis' and 'php7.3-xdebug'?   The notes on SYS-108 led me to believe they might cause the conflict?